monero-outreach-docs/translations/es/storyboard/refuge_es.md

```diff
@@ -0,0 +1,94 @@
+
+Con respecto a las injustas leyes de decomiso civil de los EEUU (y a una [minoría de jurisdicciones](https://en.wikipedia.org/wiki/Asset_forfeiture#European_Union)), Monero sirve para prevenir lo siguiente: 
+
+1. Comportamiento codicioso del gobierno y de la policía.
+2. Castigos arbitrarios y/o desproporcionados. 
+
+Veamos cómo Monero puede ayudar a prevenir ambos. 
+
+### **Comportamiento policial y gubernamental codicioso** 
+
+Como señalamos sobre el programa norteamericano de "intercambio equitativo", empleado por algunos estados, el sistema incentiva el comportamiento abusivo y codicioso policíaco, permitiendo la operación libre de impunidad de parte de las agencias gubernamentales y funcionarios del gobierno. Activos líquidos como el dinero, joyas, vehículos y casas, ofrecen un gancho tentador para los funcionarios, aumentando la presunción de actividad ilegal de parte de la policía. El dinero en efectivo y las joyas son fácilmente convertidos en instrumentos y herramientas policiales. 
+
+Además, la mayoría de estados y el gobierno federal autorizan por ley que las agencias y funcionarios [mantengan un 90% de los beneficios](https://www.forbes.com/2011/06/08/property-civil-forfeiture.html#2f2b943439fa) de los activos incautados, un claro motivo de lucro que induce al principio de decomiso.
+
+*Pero, ¿que sucedería si los activos y el dinero del individuo pudieran mantenerse en formato digital?* 
+
+Si no hay un activo tangible a plena vista para un agente de policía o un funcionario, la probabilidad de registro y su posible incautación disminuyen significativamente. 
+
+Ten en cuenta a Monero y a otras criptomonedas que preservan la privacidad. Mientras que la mayoría de las criptomonedas de persona a persona (p2p) están en el camino correcto hacia la autonomía de la codicia y la aplicación de las leyes de decomiso, aún no son inmunes. Han sido monitoreadas y rastreadas en cierto grado por [empresas privadas y agencias gubernamentales a través de los UTXOs (salida de transacción no gastada)](https://bitcoinist.com/yes-your-bitcoin-transactions-can-be-tracked-and-here-are-the-companies-that-are-doing-it/). 
+ 
+Sin embrago, Monero a través del [protocolo CryptoNote](https://cryptonote.org/) opera con diferencias muy marcadas (en comparación con Bitcoin) en la manera de ocultar información en la cadena de bloques. Mediante firmas de círculo se ocultan las entradas de una transacción entre otros múltiples posibles entradas, incrementando la dificultad de vínculos con transacciones seguidas. Mientras, [Malte Möser de la universidad de Princeton destacó que el 62% las transacciones](https://www.technologyreview.com/s/610807/sitting-with-the-cyber-sleuths-who-track-cryptocurrency-criminals/) de Monero fueron vulnerables al análisis de rastreo (abril del 2018), no fue por deficiencias en el protocolo sino por errores humanos y prácticas de privacidad descuidadas. 

```

Comment:
Wasn't sure if this was the other I found but it needs to be changed:

+ _**Sin embargo**_, Monero a través del...

Reply:
Oh I think I missed it and marked it before as done. Thank you for the notice @BlackLotus64. Fixed >:).